android/src/main/java/com/imagepicker/utils/MediaUtils.java

```diff
@@ -66,6 +66,35 @@
         return result;
     }
 
+    public static @Nullable File createNewVideoFile(@NonNull final Context reactContext,
```

Comment:
Ideally, createNewFile and createNewVideoFile would be combined, but this is a quick fix.

Reply:
Actually, just did this